lib/fontcustom/scripts/generate.py

```diff
@@ -46,6 +46,30 @@
 # Glyphs
 #
 
+def resolveGlyphName( c ):
+    if c == "0":
+        c = "zero"
+    elif c == "1":
+        c = "one"
+    elif c == "2":
+        c = "two"
+    elif c == "3":
+        c = "three"
+    elif c == "4":
+        c = "four"
+    elif c == "5":
+        c = "five"
+    elif c == "6":
+        c = "six"
+    elif c == "7":
+        c = "seven"
+    elif c == "8":
+        c = "eight"
+    elif c == "9":
+        c = "nine"
```

Comment:
Not a huge fan of this approach. Consider using an array.
`
words = ['zero', 'one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine']
return words[c] || c
`